infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/KubernetesServerExposer.java

```diff
@@ -130,24 +104,19 @@
   public static final int SERVER_UNIQUE_PART_SIZE = 8;
   public static final String SERVER_PREFIX = "server";
 
-  private final Map<String, String> ingressAnnotations;
+  protected final ExternalServerExposerStrategy kubernetesExternalServerExposerStrategy;
```

Comment:
Could you generify this field with `<T>` to avoid unchecked call here https://github.com/eclipse/che/pull/8822/files#diff-ff394ec17243637d9571b51b4aa23b99R206

Reply:
done